sdk/containerapps/Azure.ResourceManager.AppContainers/src/Generated/Models/ServiceBind.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.AppContainers.Models
+{
+    /// <summary> Configuration to bind a ContainerApp to a dev ContainerApp Service. </summary>
+    public partial class ServiceBind
+    {
+        /// <summary> Initializes a new instance of ServiceBind. </summary>
+        public ServiceBind()
+        {
+        }
+
+        /// <summary> Initializes a new instance of ServiceBind. </summary>
+        /// <param name="serviceId"> Resource id of the target service. </param>
+        /// <param name="name"> Name of the service bind. </param>
+        internal ServiceBind(string serviceId, string name)
+        {
+            ServiceId = serviceId;
+            Name = name;
+        }
+
+        /// <summary> Resource id of the target service. </summary>
+        public string ServiceId { get; set; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#d4cebe383870410cab2e14d0709f78bb).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#9f6fc0ae7c8e497e98d7979dadf97a4c).

Reply:
As mentioned in summary, it is resource id